public/locales/de/translations.json

```diff
@@ -0,0 +1,49 @@
+{
+	"root.description": "Eine Alternative zu dem Nix Ökosystem",
+
+	"header.community": "Gemeinschaft",
+
+	"goals.title": "Die Ziele",
+	"goals.independent.title": "Unabhängigkeit",
+	"goals.independent": "We will fork and maintain Nix, NixPkgs, and NixOS. Additional support for projects such as Nix Darwin and Home Manager will be considered.",
+	"goals.gov.title": "Gesellschaftliche Organisation",
+	"goals.gov": "A democratic model of governance with elected positions will be used to manage the project. A steering committee will provide direction with additional committees handling specific logistical tasks. Features or individual projects will be owned by Special Interest Groups. SIGs will collaborate within Working Groups to achieve their goals. Each committee and group will have its goals, requirements, and management process clearly defined.",
+	"goals.stabilization.title": "Stabilisierung",
+	"goals.stabilization": "User-friendly features such as the Nix v3 CLI will be enabled by default. Nix Flakes are used too heavily to remove or change and will instead be stabilized as a v0 in its current state with any future work being handled by a Special Interest Group.",
+	"goals.infra.title": "Infrastruktur",
+	"goals.infra": "We will build and maintain the tools and infrastructure necessary to keep the ecosystem healthy, including Continuous Integration and Binary Caching.",
+	"goals.education.title": "Dokumentation",
+	"goals.education": "Unified documentation, tutorials, guides, and examples will provide a better onboarding experience for newcomers and a faster iteration cycle for experts.",
+
+	"roadmap.title": "Der Fahrplan",
+	"roadmap.soft_fork.title": "Soft Fork",
```

Comment:
@liketechnik I'm not sure it makes sense to translate soft/hard fork. The terms feel super specific to computer lingo, I can't think of a good alternative.

Any ideas?

Reply:
I'd also just leave it at soft/hard fork, it's an established term in technology and really does not translate well. 
Generally in German, you often just use the English terms anyway for simplicity, esp. for such specificities.

Reply:
Not really, no. 

But I guess it's also fine, 'Fork'/'Forking' seems to be pretty widely used in technical documentation in German, according to my quick google search (e.g. GitHub's German docs use 'Forking').

Reply:
ACK, will resolve this convo since we're in agreement